src/templates/app/resources/economic_calendar.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const EconomicCalendar = () => (
+    <div id='economic_calendar' className='static_full'>
+        <div className='economic-calendar-header gr-row'>
+            <div className='gr-12 gr-centered'>
+                <h1>{it.L('Economic Calendar')}</h1>
+                <p>{it.L('Check out the latest economic events and indicators from all over the world. All data are collected and updated in real time.')}</p>
+            </div>
+        </div>
+        <div className='gr-12 gr-centered'>
```

Comment:
why do we need the extra padding here and on line 16? it makes it smaller than the first paragraph